includes/Emails/FakeMailer.php

```diff
@@ -0,0 +1,13 @@
+<?php
```

Comment:
kindly remove this class and revert to old code for this section. This code already been rewritten on HPOS pr